src/services/models/iotHubManagerModels.js

```diff
@@ -24,7 +24,7 @@ export const toDeviceModel = (device = {}) => {
     'c2DMessageCount': 'c2DMessageCount',
     'enabled': 'enabled',
     'lastStatusUpdated': 'lastStatusUpdated',
-    'ioTHubHostName': 'ioTHubHostName',
+    'ioTHubHostName': 'iotHubHostName',
```

Comment:
seems strange that the 't' is capital case on the left, lower case on the right. Could we use the same format?

Reply:
Here is what is happening:
From IotHubManager we get IoTHubHostName
The UI auto-camelcases everything, which turns into ioTHubHostName
we want it to be iotHubHostName in the code, so this maps ioTHubHostName coming from our request to iotHubHostName, what we want the variable to be called.

Reply:
thanks! that would be a great comment to add future readers